pkg/kn/commands/service_get_test.go

```diff
@@ -90,11 +165,17 @@ func testContains(t *testing.T, output string, sub []string, element string) {
 	}
 }
 
-func createMockServiceWithParams(name, domain string, generation int64) *v1alpha1.Service {
+func createMockServiceListWithParams(name, domain string, generation int64, version bool) *v1alpha1.Service {
+	var apiVersion string
+	if version {
```

Comment:
see comment in revision get

Reply:
https://github.com/knative/client/pull/150#discussion_r289274186